cli/modelsvalidator/validate.py

```diff
@@ -0,0 +1,374 @@
+import json
+import re
+import sys
+from typing import Any, Dict, List, Optional, Tuple, Union, cast
+
+import yaml
+
+DEFAULT_FILES = [
+    "./docs/models.yml",
+]
+
+
+KEYSEPARATOR = "/"
+_collection_regex = r"[a-z]([a-z_]+[a-z]+)?"
+_field_regex = r"[a-z][a-z0-9_]*\$?[a-z0-9_]*"
+
+collectionfield_regex = re.compile(
+    f"^({_collection_regex}){KEYSEPARATOR}({_field_regex})$"
+)
+collection_regex = re.compile(f"^{_collection_regex}$")
+field_regex = re.compile(f"^{_field_regex}$")
+
+decimal_regex = re.compile("^\d+\.\d{6}$")  # NOQA
+color_regex = re.compile("^#[0-9a-f]{6}$")
```

Comment:
Since this file now lives in the backend, you should be able to import some of these from the "original" files instead of duplicating the definitions here (if that's possible)

Reply:
done, except last two